src/Network/Network/help/New-AzPublicIpAddress.md

```diff
@@ -14,7 +14,7 @@ Creates a public IP address.
 ## SYNTAX
 
 ```
-New-AzPublicIpAddress [-Name <String>] -ResourceGroupName <String> [-Location <String>] [-Sku <String>]
+New-AzPublicIpAddress -Name <String> -ResourceGroupName <String> -Location <String> [-Sku <String>]
```

Comment:
This change cannot be kade in the help file.  It must be made in cmdlet code.  The syntax portion of help is generated, so this change will be wiped out the next time help is generated.  Additionally, PowerShell users will only gain the benefits of the parameter being mandatory (like being prompted if the parameter is not provided)

Reply:
The parameter is already mandatory in the sense that the user receives an error if they don't supply it. However, the help file/documentation doesn't specify it is required. Under **Parameters** in the help file, I changed **Required** to be **True** for these. Is this incorrect? 

Thanks!